Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /tikv
 
 # Install Rust
 COPY rust-toolchain ./
-RUN rustup default nightly-2019-04-25
+RUN rustup default nightly-2019-06-14
```

Comment:
Is this line required? rustup can choose the correct toolchain based on the `rust-toolchain` file.

Reply:
we need to cache the Rust explicitly at first here.